cmd/crio/config.go

```diff
@@ -45,6 +45,9 @@ file_locking = {{ .FileLocking }}
 # Path to the lock file.
 file_locking_path = "{{ .FileLockingPath }}"
 
+# Whether CRI-O should wipe storage after a CRI-O upgrade happens
+auto_upgrade = {{ .AutoUpgrade }}
```

Comment:
I feel like `auto_upgrade` may not be the best choice for the option name here.
How about something like `clear_storage_on_upgrade` ?

Reply:
kind of a mouthful but I can't think of anything better. sounds good